### PR TITLE
Apt speedup

### PR DESCRIPTION
**Pull request by swbsf**
Some little tweaks to speed up long processes on ARM CPUs.

### CODE REVIEW

File: roles/system/tasks/main.yml [hunk 1]
```diff
@@ -17,14 +17,14 @@
...
-  register: result
+- name: Update APT cache, not more than once every 8 hours
+  apt: 
+    update_cache: yes 
+    cache_valid_time: 28800
   tags: ['master','custom','update']
 
 - name: Fix broken package
   shell: dpkg --configure -a
-  when: "'dpkg --configure -a' in result.stderr"
```
Comment (fheslouin) [inline roles/system/tasks/main.yml:27]: Is there a reason to play this task at every run ? 
`when: "'dpkg --configure -a' in result.stderr"` was there to play this task only when `shell: apt-get update ; echo` crashed

Comment (swbsf) [reply roles/system/tasks/main.yml:27]: Yes, this is a choice to use apt module. I didn't look at apt module output to see if behavior is the same as previous apt-cache update command. Even if this is highly probable.
I decided the new behavior (running the task each time) is not really an issue as the command is very fast when there's nothing to do.

Comment (fheslouin) [reply roles/system/tasks/main.yml:27]: This is an option indeed.

**Comment by fheslouin** [comment]
Can we merge that one ?

**Pull request merged**